### PR TITLE
DOC: Ambiguous description in to_parquet engine documentation 

### LINKED ISSUE #19662: Ambiguous description in pandas.DataFrame.to_parquet documentation
#### Relevant Docstring

In the verson 0.22.0 [pandas.DataFrame.to_parquet documentation](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_parquet.html), the **engine** kwarg is described as:
>"Parquet reader library to use. If ‘auto’, then the option ‘io.parquet.engine’ is used. If ‘auto’, then the first library to be installed is used."

The two descriptions of "auto" are in conflict, so the one that isn't correct should be removed.

### PR DESCRIPTION
**Pull request by giba0**
Ambiguity correction in documentation on pandas.DataFrame.to_parquet

- [x] closes #19662
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] tests added / passed
- [x] whatsnew entry

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Looks like some stray commits got into your PR. Do you need help cleaning up your branch?

**Comment by gilbertoolimpio** [comment]
yes I need! I tried in several ways to clear the jump but I could not! can you help me?

**Comment by TomAugspurger** [comment]
Ok, should be good now. I added a commit deleting your window-related changes.

In general, when you make a PR you want to

```
# fetch the latest changes from master
git fetch upstream

git checkout master
git pull

# Checkout a new branch for your changes
git checkout -b parquet-docs
```

And you do that for each PR independently.


1. Fetch the latest changes from master

**Comment by gilbertoolimpio** [comment]
@tomauguspurger thanks for helping this newbie!

File: pandas/core/frame.py [hunk 1]
```diff
@@ -1678,9 +1678,10 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
         fname : str
             string file path
         engine : {'auto', 'pyarrow', 'fastparquet'}, default 'auto'
-            Parquet reader library to use. If 'auto', then the option
-            'io.parquet.engine' is used. If 'auto', then the first
-            library to be installed is used.
+            Parquet library to use. If 'auto', the value from
+            ``io.parquet.engine`` is used. The default ``io.parquet.engine``
```
Comment (jorisvandenbossche) [inline pandas/core/frame.py]: I would clarify "io.parquet.engine", as now it seems like a constant in pandas.io.parquet.engine, which is not the case, it is an option. Maybe leaving "*the option* io.parquet.engine .." as before is enough

**Comment by gilbertoolimpio** [comment]
@jorisvandenbossche Ops! now was better? =)

**Comment by jorisvandenbossche** [comment]
Yes, that's better. Can you do the same for the other occurrences?

**Comment by gilbertoolimpio** [comment]
@jorisvandenbossche newbie here! =) Thanks for your help! Now all updated.

**Review by jorisvandenbossche — APPROVED** [review]
Yes, looking good now!

**Pull request merged**